app/locales/en.js

```diff
@@ -185,5 +185,14 @@ export default {
       section: 'Project email preferences',
       header: 'Project'
     }
+  },
+  about: {
+    home: {
+      title: '## What is the Zooniverse?',
+      whatIsZooniverse: 'The Zooniverse is the world’s largest and most popular platform for people-powered research. This research is made possible by volunteers — hundreds of thousands of people around the world who come together to assist professional researchers. Our goal is to enable research that would not be possible, or practical, otherwise. Zooniverse research results in new discoveries, datasets useful to the wider research community, and [many publications](/about/publications).',
+      anyoneCanResearch: '### At the Zooniverse, anyone can be a researcher\n\nYou don’t need any specialised background, training, or expertise to participate in any Zooniverse projects. We make it easy for anyone to contribute to real academic research, on their own computer, at their own convenience.\n\nYou’ll be able to study authentic objects of interest gathered by researchers, like images of faraway galaxies, historical records and diaries, or videos of animals in their natural habitats. By answering simple questions about them, you’ll help contribute to our understanding of our world, our history, our Universe, and more.\n\nWith our wide-ranging and ever-expanding suite of projects, covering many disciplines and topics across the sciences and humanities, there\'s a place for anyone and everyone to explore, learn and have fun in the Zooniverse. To volunteer with us, just go to the [Projects](/projects) page, choose one you like the look of, and get started.',
```

Comment:
Instead of using newline characters, you can create multi-line strings using backticks:

```
`
Paragraph 1

Paragraph 2
`
```

Reply:
Using the backticks doesn't work with our Markdown components though, which we're using to render these sections. If we stop using Markdown, will we need to introduce our own styles if we want to keep the look of the site the same across all these static pages (see also, eg, privacy and security policy). Happy to entertain other possibilities, but not sure if using backticks/no Markdown is the easiest way to get what we want here.

![image](https://user-images.githubusercontent.com/12473119/33435340-93220c3c-d5a7-11e7-8fe9-533ca5e2160b.png)


Backticks render like this ^^

![image](https://user-images.githubusercontent.com/12473119/33435375-a02b3c28-d5a7-11e7-92f2-a206f7c63265.png)


Markdown elements render like this ^^

Reply:
Not suggesting not using Markdown, but I didn't know the markdown components couldn't handle backticks. ¯\_(ツ)_/¯

Reply:
By the way - did you use the backticks instead of the single quotes around the strings, or inside?

Reply:
Instead of. Using them inside fails to compile.

Reply:
That makes sense because that's telling the javascript to use a template literal by using back ticks.

Reply:
Since these are the strings eventually translator volunteers may see we may want to just break it up into more key/value pairs. This is probably fine for now though.